ui/src/main/resources/Meeting/MeetingSheet.xml

```diff
@@ -317,9 +253,10 @@ $xwiki.jsx.use('Meeting.MeetingSheet')
         : $doc.display('groups')
       #end
       #if($doc.isCurrentUserCreator() &amp;&amp; $doc.getValue('participants')!='' &amp;&amp; $xcontext.action=='view')
-        &lt;form id="sendNotification" action='' method='post'&gt;
+        &lt;form id="sendNotification" action="$xwiki.getURL('MeetingCode.SendInvitationEmail', 'get', 'outputSyntax=plain')" method='post'&gt;
           &lt;div class='presentation'&gt;
             &lt;input type='hidden' name='sendNotification' value='true'&gt;
+            &lt;input type='hidden' name='doc' value="${escapetool.xml($doc)}"&gt;
```

Comment:
You should not rely on the `toString()` result of `$doc` (the `Document` class) if what you really wanted was the serialized document reference. This will work, but you have no guarantee at the API level that it will not change in the future and break your code.

The safe way would be `$services.model.serialize($doc.documentReference, 'default')`.